This should fix a recent regression in JSC CLoop on BE/32bit.

--- a/Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm
+++ b/Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm
@@ -425,7 +425,7 @@ end)
 op(llint_get_host_call_return_value, macro ()
     functionPrologue()
     pushCalleeSaves()
-    loadp Callee[cfr], t0
+    loadp Callee + PayloadOffset[cfr], t0
     convertCalleeToVM(t0)
     loadi VM::encodedHostCallReturnValue + TagOffset[t0], t1
     loadi VM::encodedHostCallReturnValue + PayloadOffset[t0], t0
